src/esp/sensor/FisheyeSensor.cpp

```diff
@@ -48,6 +50,14 @@ void specSanityCheck(FisheyeSensorSpec* spec) {
   actualSpec->sanityCheck();
 }
 
+Magnum::Vector2 computePrincipalPointOffset(const FisheyeSensorSpec& spec) {
+  if (spec.principalPointOffset != Corrade::Containers::NullOpt) {
```

Comment:
I think you can just use: `Corrade::Containers::Optional<T>::operator bool() const explicit`
so if (spec.principalPointOffset)